crates/voicevox_core_python_api/src/lib.rs

```diff
@@ -361,6 +371,75 @@ impl Synthesizer {
     }
 }
 
+#[pyclass]
+#[derive(Debug, Clone)]
+struct UserDict {
+    dict: voicevox_core::UserDict,
+}
+
+#[pymethods]
+impl UserDict {
+    #[new]
+    fn new() -> Self {
+        Self {
+            dict: voicevox_core::UserDict::new(),
+        }
```

Comment:
この`new`は`#[pymethods]`じゃなかったら[`clippy::new_without_default`](https://rust-lang.github.io/rust-clippy/master/index.html#new_without_default)が発動していると思うので、こうするのはどうでしょうか。

```suggestion
#[derive(Default, Debug, Clone)]
struct UserDict {
    dict: voicevox_core::UserDict,
}

#[pymethods]
impl UserDict {
    #[new]
    fn new() -> Self {
        Self::default()
```